components/expenses/UnapproveExpenseBtn.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { graphql } from 'react-apollo';
+import { FormattedMessage } from 'react-intl';
+
+import StyledButton from '../StyledButton';
+
+const UnapproveExpenseBtn = ({ id, unapproveExpense }) => {
+  const handleOnClick = async () => {
+    try {
+      await unapproveExpense(id);
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  return (
+    <StyledButton mr={2} buttonStyle="secondary" onClick={() => handleOnClick()}>
+      <FormattedMessage id="expense.unapprove.btn" defaultMessage="Unapprove" />
+    </StyledButton>
+  );
+};
+
+UnapproveExpenseBtn.propTypes = {
+  id: PropTypes.number.isRequired,
+  unapproveExpense: PropTypes.func.isRequired,
+};
+
+const unapproveExpenseQuery = gql`
+  mutation unapproveExpense($id: Int!) {
+    unapproveExpense(id: $id) {
+      id
+      status
+    }
+  }
+`;
+
+const addMutation = graphql(unapproveExpenseQuery, {
```

Comment:
For future components like this one check the newly introduced `useMutation` hook (we haven't used it yet in the codebase), it gives you a nice API do to the same in fewer lines, and you get error handling and loading indicator for free: https://www.apollographql.com/docs/react/api/react-hooks/#example-2